docker/config.properties

```diff
@@ -3,6 +3,7 @@ model_store=/opt/ml/model
 load_models=ALL
 inference_address=http://0.0.0.0:8080
 management_address=http://0.0.0.0:8081
+preload_model=true
```

Comment:
I think we should have this as false for default at least until we're sure it's stable.

Reply:
agree